recipes/cobra/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "cobra" %}
+{% set version = "0.20.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cobra-{{ version }}.tar.gz
+  sha256: db3d8df2eab62b2aa2db75e8e85811c83b248071abf208c765f8f455cfaa88e2
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - appdirs
+    - depinfo
+    - diskcache
+    - future
+    - httpx
+    - importlib_resources
+    - numpy >=1.13
+    - optlang >=1.4.2,<1.4.6
+    - pandas >=1.0
+    - pydantic
+    - python >=3.6
+    - python-libsbml-experimental ==5.18.3
```

Comment:
Does this need to be tightly pinned to this version?

Reply:
It needs to be in setup.py since there are no binary wheelsfor some of the newer versions, however, for conda this should not be necessary.